R/visualization/plotBestMethodsRNA.R

```diff
@@ -1,25 +1,58 @@
+#' Plotting scib-metrics result of the integration across multiple RNA tasks in an
```

Comment:
First line is a title so usually a few words like "Plot scIB metrics" in this case. This isn't for an actual package though so just a note for future reference.